demisto_sdk/commands/common/tests/constants_regex_test.py

```diff
@@ -84,6 +130,36 @@ def test_script_code_files():
     )
 
 
+def test_new_classifier_files():
+    acceptable_script_code_files, unacceptable_script_code_files = get_test_new_classifiers_paths(CLASSIFIERS_DIR)
```

Comment:
```suggestion
    acceptable_classifiers_files, unacceptable_classifiers_files = get_test_new_classifiers_paths(CLASSIFIERS_DIR)
```

same change in all tests

Reply:
done